### PR TITLE
add authoring guidelines

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : https://github.com/csstools/postcss-plugins/issues/614

This formalises core principles which we have adopted in the past few months.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ instructions:
 ### Quick start
 
 1. fork the repo and clone it locally
-2. `npm run get-me-going` to install and build dependencies
+2. `npm run setup-workspace` to install and build dependencies
```
Comment (romainmenke) [inline CONTRIBUTING.md:29]: This wording is a bit more clear I think

File: AUTHORING_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Authoring guidelines
+
+We want to make it easier to write awesome CSS that works great in as many browsers as possible.
+Some of these browsers are a bit older, others don't yet exist.
+
+We will not place undue burden on anyone writing or maintaining CSS.
+
+We will only add a plugin to `postcss-preset-env` when the CSS WG invites implementations for the relevant features.
+
+We will not create plugins with features that look and feel native but don't have a specification.
```
Comment (Antonio-Laguna) [inline AUTHORING_GUIDELINES.md:10]:
```suggestion
We will not create plugins with features (or add features to plugins) that look and feel native but don't have a specification.
```

Feel free to tweak the wording, an example would be #671

File: CONTRIBUTING.md (hunk 1 above)
Comment (Antonio-Laguna) [reply CONTRIBUTING.md:29]: Agreed, naming things is hard ™

Thanks for coming up with this!

**Review by Antonio-Laguna — APPROVED** [review]
This LGTM! Feel free to discard my comment if you think it's already covered. I just wanted to be extra explicit about features too.

Thanks for adding this!

File: AUTHORING_GUIDELINES.md (hunk 2 above)
Comment (romainmenke) [reply AUTHORING_GUIDELINES.md]: I agree, good catch :)

It is something we have seen in the past.
That small productivity features find their way into plugins that intended to "polyfill" native features.

Best to keep those separate.

**Pull request merged**